README.md

```diff
@@ -1 +1,102 @@
-This a webpack boilerplate for my projects
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+
+  <h3><b>LeaderBoard</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[LeaderBoard \] ](#-leaderboard--)
+  - [🛠 Built With **HTML, CSS and JavaScript**](#-built-with-html-css-and-javascript)
+    - [Tech Stack **HTML, CSS and JavaScript**](#tech-stack-html-css-and-javascript)
+    - [Key Features ](#key-features-)
+    - [LiveDemo](#livedemo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Project\_presentation](#project_presentation)
+  - [🔭 Future Features ](#-future-features-)
+  - [👥 Authors ](#-authors-)
+  - [Contributions](#contributions)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [LeaderBoard ] <a name="about-project"></a>
+**LeaderBoard** is a project under microverse Module 2 block 2 aimed at testing my understanding of ES6 syntax, modules, APIs, gitflow and webpack. This app takes and stores scores of different players playing a game.
+## 🛠 Built With **HTML, CSS and JavaScript**
+
+### Tech Stack **HTML, CSS and JavaScript**
+### Key Features <a name="key-features"></a>
+- **Contains a LeaderBoard project that takes and stores game scores for players items as ojects in an array**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### LiveDemo
+[Live Demo] You can see the live Demo of the website here  
+
```

Comment:
- [ ] Your project currently does not have a live demo, you are either required to add the correct link to the live demo or remove this section. 👍 
- Make sure that if you are removing this section you also remove from the header links above in your readme file 👍 ✨